server/routes/signup.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const router = express.Router();
+const bcrypt = require("bcrypt");
+const saltRounds = 10;
+const User = require("../models/User");
+
+router.post("/:email/:password", (req, res, next) => {
+  const { email, password } = req.params;
+  let epCheck = false;
+  if (email.includes("@") && password.length > 6 && (epCheck = true)) {
+    bcrypt.hash(password, saltRounds, (err, hash) => {
+      const addUser = async () => {
+        const user1 = new User({
+          email: email,
+          password: hash,
+        });
+        await user1
+          .save()
+          .then(() => res.status(200).send({ msg: "Account created!." }))
+          .catch((err) => {
+            res.status(401).send({ msg: err });
+          });
+      };
+      addUser();
```

Comment:
It's weird to define the function right above and then call it here once, would be better to just keep it all in the same block